shynet/analytics/views/ingress.py

```diff
@@ -29,6 +29,9 @@ def ingress(request, service_uuid, identifier, tracker, payload):
     location = request.META.get("HTTP_REFERER", "").strip()
     user_agent = request.META.get("HTTP_USER_AGENT", "").strip()
     dnt = request.META.get("HTTP_DNT", "0").strip() == "1"
+    gpc = request.META.get("HTTP_Sec-GPC", "0").strip() == "1"
```

Comment:
I *think* it would behave fine all capitalised as well. I did it that way to be verbatim with the spec, but I don't think it should cause any issues otherwise.

Reply:
Rephrase: it will be fine. RFC 2616 (or whatever's newer) does say that header names aren't case sensitive. I'll switch it to all caps, and it should be dandy.